charts/palworld/values.yaml

```diff
@@ -26,7 +26,14 @@ server:
     # -- The size of the pvc storage.
     size: 12Gi
     # -- The storage class name.
-    storageClassName: ""
+    ## data Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClassName>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    ##
+    # storageClassName: "-"
```

Comment:
Explain this to me: why would you ever want to explicilty set the `storageClassName` to an empty string?

AFAIK If no name is explicitly set then kubernetes will infer the default storage class. Wouldn't make much sense to explicitly set it to an empty string.